projects/packages/analyzer/scripts/jetpack-slurper.php

```diff
@@ -57,6 +58,9 @@
 	$warnings->generate( $invocations, $differences );
 	$warnings->output();
 	$warnings->save( __DIR__ . '/' . basename( $folder_name ) . '.csv', false );
+
+	unset( $invocations );
+	unset( $warnings );
```

Comment:
Feel free to ignore, but a comment to explain why you're explicitly unsetting might be nice.
```suggestion
	// Faster to GC all the data before the next iteration.
	unset( $invocations, $warnings );
```